tests/completion/snippets/task.yaml

```diff
@@ -1,4 +1,25 @@
-summary: indirect completion
+summary: snippets completion?
```

Comment:
```suggestion
summary: snippets completion
```

Reply:
I left the question mark intact as I don't really understand what kind of snippets are being mentioned here.

Reply:
I see this test is almost the same than hte indirect one
perhaps it is not adding any value